Idno/Data/MySQL.php

```diff
@@ -75,6 +75,7 @@ function checkAndUpgradeSchema()
                             2017032001,
                             2019060501,
                             2019121401,
+                            2020042101,
```

Comment:
is this one supposed to update too?

Reply:
Only if you're running with the mysql engine. 

Scripts only execute for the engine you're on.

Reply:
Right. Well the postgres edits are not in-line with the MySQL ones then. It's one of the parts that was confusing.

Reply:
Well, some patches are only relevant to specific database engines hence why the difference (also, Postgres isn't so much a well trodden path)